linux_os/guide/services/ntp/service_timesyncd_configured/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Configure Systemd Timesyncd Servers'
+
+description: |-
+    <tt>systemd-timesyncd</tt> is a daemon that has been added for synchronizing the system clock
+    across the network. More information on <tt>chrony</tt> can be found at
```

Comment:
This "More information on <tt>chrony</tt> can be found at" seems that it shouldn't be here.

Reply:
Thanks, reworked in ad98bb7 obviously too much copy paste :man_facepalming: